invenio_migrator/records.py

```diff
@@ -49,6 +49,19 @@ class RecordDumpLoader(object):
     @classmethod
     def create(cls, dump):
         """Create record based on dump."""
+        # If 'record' is not present, just create the PID
+        if not dump.data.get('record'):
+            try:
+                PersistentIdentifier.get(pid_type='recid',
```

Comment:
Sorry, I actually have to use the PI.create instead of provider since I am providing the recid as ``pid_value``, which is forbidden in the provider: https://github.com/inveniosoftware/invenio-pidstore/blob/master/invenio_pidstore/providers/recordid.py#L53